include/SFML/Network/Ftp.hpp

```diff
@@ -484,7 +484,7 @@ class SFML_NETWORK_API Ftp : NonCopyable
     /// \see download
     ///
     ////////////////////////////////////////////////////////////
-    Response upload(const std::string& localFile, const std::string& remotePath, TransferMode mode = Binary);
+    Response upload(const std::string& localFile, const std::string& remotePath, TransferMode mode = Binary, const append = false);
```

Comment:
I've compiled it on my machine, but I've never made a PULL REQUEST, I did it quiquly without uploading my real SFML project code, I'll modify this, but if you find another error, please just do it as you think it could be. Thanks